src/main/scala/io/druid/indexer/spark/SparkDruidIndexer.scala

```diff
@@ -424,18 +423,18 @@ object SparkDruidIndexer {
 
 object SerializedJsonStatic {
   val LOG = new Logger("io.druid.indexer.spark.SerializedJsonStatic")
+  val defaultService = "spark-indexer"
+  // default indexing service port
+  val defaultPort = "8090"
   lazy val injector: Injector = {
     try {
       Initialization.makeInjectorWithModules(
         GuiceInjectors.makeStartupInjector(), List[Module](
           new Module {
             override def configure(binder: Binder): Unit = {
-              JsonConfigProvider
-                .bindInstance(
-                  binder,
-                  Key.get(classOf[DruidNode], classOf[Self]),
-                  new DruidNode("spark-indexer", null, null)
-                )
+              binder.bindConstant().annotatedWith(Names.named("serviceName")).to(defaultService)
```

Comment:
`DruidNode` uses `@JacksonInject` but the class doesn't allow null values:
```
 @JsonProperty("service")
  @NotNull
  private String serviceName;

  @JsonProperty
  @NotNull
  private String host;
```
My intention was to make the service configurable so I can configure the `service` dimension that the `ServiceEmitter` uses. I also don't like having those default values, but I don't think I can use `@JacksonInject` for `DruidNode` unless I modify `DruidNode` class or have the users to provide `-Ddruid.service...` in VM options always.